pymatgen/io/cif.py

```diff
@@ -79,6 +81,14 @@ def _get_cod_data():
     return _COD_DATA
 
 
+upper_case_elements = ['AC', 'AG', 'AL', 'AM', 'AR', 'AS', 'AT', 'AU', 'BA', 'BE', 'BI', 'BK', 'BR', 'CA',
+                       'CD', 'CE', 'CF', 'CL', 'CM', 'CO', 'CR', 'CS', 'CU', 'DY', 'ER', 'ES', 'EU', 'FE',
+                       'FM', 'FR', 'GA', 'GD', 'GE', 'HE', 'HF', 'HG', 'HO', 'IN', 'IR', 'KR', 'LA', 'LI',
+                       'LR', 'LU', 'MD', 'MG', 'MN', 'MO', 'NA', 'NB', 'ND', 'NE', 'NI', 'NO', 'NP', 'OS',
+                       'PA', 'PB', 'PD', 'PM', 'PO', 'PR', 'PT', 'PU', 'RA', 'RB', 'RE', 'RH', 'RN', 'RU',
+                       'SB', 'SC', 'SE', 'SI', 'SM', 'SN', 'SR', 'TA', 'TB', 'TC', 'TE', 'TH', 'TI', 'TL',
+                       'TM', 'XE', 'YB', 'ZN', 'ZR']
```

Comment:
Do we really need to specify upper case elements? Can't we simply check Element.is_valid_symbol(string.title())?

Reply:
I agree that the listing of all the symbol is far from an elegant solution (notice that only two letters elements are liste here). Unfortunately there are cases that have labels like "MgT" (e.g. http://www.crystallography.net/cod/view.php?f=9003921.cif , but there are several similar examples). In such cases Element.is_valid_symbol(string.title()) will fail. 
Since in the _atom_site_label one can apparently find all sort of notations and symbols, I ultimately decided to go for the explicit matching with the elements. Any other suggestion is welcome.

Reply:
I think you should basically just stipulate that you parse the first two letters, and title case them. Of course, all these should come with a warning. I am just not a fan of writing this kind of specific conversions which is very fragile and not general.